homeassistant/components/wled/light.py

```diff
@@ -96,14 +93,16 @@ async def async_setup_entry(
 class WLEDMasterLight(WLEDEntity, LightEntity):
     """Defines a WLED master light."""
 
-    _attr_supported_features = SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION
+    _attr_color_mode = COLOR_MODE_BRIGHTNESS
```

Comment:
```suggestion
    _attr_color_mode = COLOR_MODE_BRIGHTNESS
    _attr_supported_color_modes = {COLOR_MODE_BRIGHTNESS}
```

Reply:
We cannot set a mutable value as a default in a class variable

Reply:
Why not? Are you worried it will be updated in an instance?